src/qat/purr/compiler/devices.py

```diff
@@ -713,7 +716,7 @@ def __init__(
 
         self.pulse_hw_x_pi_2 = {
             "shape": PulseShapeType.GAUSSIAN,
-            "width": 100e-9,
+            "width": 9.6e-08,
```

Comment:
why is this change ?

Reply:
To respect with HW specific granularity. Since I will be moving this feature internally, this change is no longer necessary.